frontend/benefit/applicant/src/constants.ts

```diff
@@ -12,6 +12,7 @@ export enum ROUTES {
   APPLICATION_FORM = '/application',
   LOGIN = '/login',
   TERMS_OF_SERVICE = '/terms-of-service',
+  ACCESSIBILITY_STATEMENT = '/accessibility-statement',
```

Comment:
On a side note, should these be Finnish by default? Then maybe translate them later?

Reply:
We have a URL schema which includes the language part, e.g. `https://helsinkilisa.hel.fi/fi/login` and `https://helsinkilisa.hel.fi/en/login`, which seems to be common practice in other Helsinki sites. Another common pattern is to translate all routes, which, I think, is a bit better, but we lack support for that currently.